packages/web-runtime/src/composables/upload/useUpload.ts

```diff
@@ -22,11 +21,15 @@ export interface UppyResource {
     currentFolder: string
     relativeFolder: string
     relativePath: string
-    route: Location
     tusEndpoint: string
     webDavBasePath: string
     uploadId: string
     topLevelFolderId?: string
+    routeName?: string
+    routeItem?: string
+    routeShareName?: string
+    routeStorage?: string
+    routeStorageId?: string
```

Comment:
For the share jail / `SharedResource.vue` we also need the `shareId` as additional param. That doesn't come from `route.params` but `route.query` though.